training_record/services.py

```diff
@@ -79,6 +79,63 @@ def create_off_campus_record_from_raw_data(
                 )
         return record
 
+    # pylint: disable=invalid-name
+    # pylint: disable=redefined-builtin
+    @staticmethod
+    def update_off_campus_record_from_raw_data(
+            record, off_campus_event=None, user=None,
```

Comment:
Even though we need to keep the variable name for `off_campus_event`, we can assign it to another variable `off_campus_event_data` right after we enter this function.